client/public/index.html

```diff
@@ -7,10 +7,29 @@
   <title>weConnect</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <!-- Import Google Icon Font -->
-  <link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <!-- Import materialize.css -->
+  <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
   <link href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.0/css/materialize.min.css" rel="stylesheet">
 
+ 
+<style>
+  body {
+    display: flex;
+    min-height: 100vh;
+    flex-direction: column;
+  }
+
+  main {
+    flex: 1 0 auto;
+  }
+
+</style>
+ <script>
+  // $(document).ready(function() {
```

Comment:
There are commented out codes here too,I understand that you want to push yourself to try out different approaches to solving a problem, but please when you find the one that works, kindly clean up by clearing out commented out codes. Good job Ann.